doc/source/whatsnew/v1.2.1.rst

```diff
@@ -36,6 +36,76 @@ Fixed regressions
 
 .. ---------------------------------------------------------------------------
 
+.. _whatsnew_121.ufunc_deprecation:
+
+Calling numpy ufuncs on non-aligned DataFrames
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before pandas 1.2.0, calling a numpy ufunc on non-aligned DataFrames (or
+DataFrame / Series combination) would ignore the indices, only match
+the inputs by shape, and use the index/rows of the first DataFrame for
```

Comment:
also for columns index

Reply:
Ah, yes, all those "index/rows" was meant to be "index/columns"